README.md

```diff
@@ -255,3 +255,14 @@ sub-dependencies. A sure way to solve it is to run:
 
     rm -fr node_modules
     yarn install
+
+### `Error: listen EADDRINUSE: address already in use :::5000`
+
+The default server port is conflicting with another running process
+(e.g., ControlCenter.app in macOS >= 12). To resolve this, you can pick any
```

Comment:
This first sentence isn't always correct. It's just an example. 
Would you mind rewriting the sentence so that it just talks about that port `:5000` might be in use by some other process and you can change it.

Reply:
I have reworded the first sentence so that it's more generic.